.eleventy.js

```diff
@@ -29,6 +30,10 @@ module.exports = eleventyConfig => {
     return moment.utc(startedAt).isAfter(moment.utc()) ? content : "";
   });
 
+  eleventyConfig.addFilter("cssmin", function(code) {
+    return new CleanCSS({}).minify(code).styles;
+  });
```

Comment:
`eleventyConfig.addFilter("cssmin", (code) => new CleanCSS({}).minify(code).styles);`

Reply:
Hi, Thank you for the feedback, I've updated the code.